tlsn/tlsn-formats/src/http/commit.rs

```diff
@@ -270,6 +278,14 @@ pub trait HttpCommit {
         direction: Direction,
         response: &Response,
     ) -> Result<(), HttpCommitError> {
+        builder.commit(response, direction).map_err(|e| {
```

Comment:
What if the HTTP response looks like:
`HTTP-Version SP Status-Code SP Reason-Phrase CRLF
CRLF`
then this line's commitment will be the same as `.without_data()` below and will cause a duplicate commitment error.